core/src/main/java/tachyon/worker/WorkerStorage.java

```diff
@@ -322,6 +324,12 @@ public WorkerStorage(InetSocketAddress masterAddress, ExecutorService executorSe
 
     mDataFolder = WorkerConf.get().DATA_FOLDER;
     mUserFolder = CommonUtils.concat(mDataFolder, WorkerConf.USER_TEMP_RELATIVE_FOLDER);
+
+    if (WorkerConf.get().EVICT_STRATEGY_TYPE.needReferenceFrequency()) {
+      mLastAttenuateBlockReferenceMs = System.nanoTime() / 1000000L;
```

Comment:
Is System.nanoTime necessary? It seems like the block reference period will not be in the milliseconds or tens of milliseconds.